rw-system.sh

```diff
@@ -198,11 +198,27 @@ if [ "$(getprop ro.vendor.product.manufacturer)" = "motorola" ] || [ "$(getprop
     fi
 fi
 
-if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/jasmine; then
+if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/jasmine -e xiaomi/jasmine_sprout; then
     setprop persist.imx376_sunny.low.lux 310
     setprop persist.imx376_sunny.light.lux 280
     setprop persist.imx376_ofilm.low.lux 310
     setprop persist.imx376_ofilm.light.lux 280
+    setprop ro.sf.lcd_density 460
+
+    # VoLTE
+    setprop persist.dbg.volte_avail_ovr 1
+
+    # Dual Camera
```

Comment:
What's the issue you're trying to fix?

Reply:
Looking back, this sector of tweaks isn't necessary.